### ISSUE TITLE
Custom title in chat-no-jquery.min.js not working

### ISSUE THREAD

**Issue by jonnitto** [zammad/zammad#5016]
### Used Zammad Version

6.2.0

### Environment

- Latest Browser (Safari, Chrome. Firefox, etc)

### Actual behaviour

If I want to set a custom title like this: 

```js
new ZammadChat({
  title: 'Foo bar'
});
```

As it seems, there is some css missing

```css
.zammad-chat-welcome {
  display: flex;
}
```

The `console.log` `chat notice || Translation needed for 'Foo bar'` is kind of misleading, as there cannot be a translation for a custom title. The same as for the option `scrollHint`. If a custom `title` or `scrollHint` is set, there should be no notice. It should also not try to translate it. 

### Expected behaviour

The title is show in the chat bar

### Steps to reproduce the behaviour

Replace `___YOUR_HOST___ ` with your zammad url

```html
<script>
    window.initZammadChat = function(){
        new ZammadChat({
            title: 'Custom title',
            chatId: 1,
            host: 'wss://___YOUR_HOST___/ws'
        })
    }
</script>
<script src="https://___YOUR_HOST___/assets/chat/chat-no-jquery.min.js" defer onload="initZammadChat()"></script>
```

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by jonnitto** [comment]
The same is with the setting `scrollHint`

**Comment by KaiSoellch** [comment]
in my case it was a missing css prop. try add this to your css
`
.zammad-chat .zammad-chat-welcome {
  display: flex;
}`

**Comment by jonnitto** [comment]
Yes, it seems like that. The console log was misleading. So the css need to be updated

**Comment by Bozo0511** [comment]
I could not recreate the issue, the chat snipet works for me as designed even when I set a custom chat title

**Comment by jonnitto** [comment]
I'll check if it is still an issue with the current version.

**Comment by jonnitto** [comment]
It's still not solved. I still need `.zammad-chat-welcome:not(.zammad-chat-is-hidden){display:flex}`

**Comment by jonnitto** [comment]
@Bozo0511 Could you re-open this?

**Comment by rolfschmidt** [comment]
I don't see the issue. Please provide more information or screenshot what's wrong. Also, it is fine that the title is translated.

![image](https://github.com/user-attachments/assets/0a5fb61e-6500-40eb-9263-ad76b3fd9b41)

```html
<html>
<body>
<script src="https://rolf.zammad.com/assets/chat/chat-no-jquery.min.js"></script>
<script>
(function() {
  new ZammadChat({
    title: 'Foo bar',
    fontSize: '12px',
    chatId: 1
  });
})();
</script>
</body>
</html>
```

**Comment by mgruner** [comment]
No feedback, closing. Please feel free to provide more information.

**Comment by jonnitto** [comment]
I can send you a screenshot where the title is not showing. I think it depends on how the chat window is embedded into the page and which CSS resets are taking place. I'm using tailwind, so this might be the source. But as a solution is already posted, why not just include this? 

In my case, I've included following style to make it work:

```css
.zammad-chat-welcome:not(.zammad-chat-is-hidden) {
   display:flex
}
```

**Comment by KaiSoellch** [comment]
We also used tailwind so i can confirm 👍
